include/llvm/IR/DerivedTypes.h

```diff
@@ -247,7 +247,7 @@ class StructType : public CompositeType {
 
   /// Try to lookup a structure type by name, and create one if one does not
   /// exist.
-  static StructType *getOrCreate(LLVMContext &Context, StringRef Name);
+  static StructType *lookupOrCreate(LLVMContext &Context, StringRef Name);
 
```

Comment:
Was this necessary?

Reply:
So I was the one who initially added `getOrCreate` to this file.  I'm confident that this method belongs in this file.  But after reading the documentation about the other `get` methods in this file, I didn't feel comfortable continuing to use `get` in the name of this method.  This change might not be strictly necessary, but I think it helps differentiate the semantics of this method from other methods with similar names.